src/views/progress.hbs

```diff
@@ -51,6 +51,18 @@
   </div>
 </article>
 
+<article class="w-100 mw7 center bb b--black-10 pb4 pt4">
+  <div class="flex justify-between flex-wrap items-center w-80 margin-0-auto">
+    <div class="pl4 pr4">
+      <h3>Additional Information</h3>
+      <h4 class="fw2 black-60">Takes 0-10 minutes</h4>
```

Comment:
Maybe we should just say it takes 5 minutes so it's consistent with the other sections

Reply:
:+1: